device-types/Dell/VxRail-P670F.yaml

```diff
@@ -0,0 +1,88 @@
+---
+manufacturer: Dell
+model: VxRail P670F
+slug: dell-vxrail-p670f
+u_height: 2
+is_full_depth: true
+airflow: front-to-rear
+weight: 35.2
+weight_unit: kg
+front_image: true
+rear_image: true
+comments: |
+  [Dell VxRail installation](https://dl.dell.com/content/manual44265077-dell-vxrail-p670f-p670n-v670f-and-s670-installation-and-service-manual.pdf?language=en-us)
+  [Dell VxRail datasheet](https://dl.dell.com/content/manual43304433-dell-vxrail-p670f-p670n-v670f-and-s670-technical-specifications.pdf?language=en-us)
+power-ports:
+  - name: Power 1
+    type: iec-60320-c14
+  - name: Power 2
+    type: iec-60320-c14
+interfaces:
+  - name: Gig-E 1
+    type: 1000base-t
+  - name: Gig-E 2
+    type: 1000base-t
+  - name: iDRAC
+    type: 1000base-t
+    mgmt_only: true
+module-bays:
+  - name: AHCI controller in SL 6
+    position: AHCI.SL.6-1
+    label: BOSS-S2
+  - name: Backplane 1 on Connector 0 of Storage Controller in SL 3
+    position: Internal.0-1:NonRAID.SL.3-1
+    label: BP15G+EXP 0:1
```

Comment:
Unfortunately storage is not suppose to be modelled in NetBox at the moment. We have an FR open to add this functionality but currently this is not supported. Please remove these definitions.